app/code/Magento/Cms/Test/Unit/Model/Wysiwyg/Images/StorageTest.php

```diff
@@ -42,9 +42,9 @@ class StorageTest extends TestCase
     /**
      * Directory paths samples
      */
-    const STORAGE_ROOT_DIR = '/storage/root/dir/';
+    private const STORAGE_ROOT_DIR = '/storage/root/dir/';
```

Comment:
Making previously public consts private is backwards incompatible. Custom code might have used these consts.
Can you make them explicitly public again instead of private? Thanks! 🙂